python/ql/lib/semmle/python/RegexTreeView.qll

```diff
@@ -48,6 +49,19 @@ newtype TRegExpParent =
   /** A back reference */
   TRegExpBackRef(Regex re, int start, int end) { re.backreference(start, end) }
 
+/**
+ * Provides regular expression patterns.
+ */
+module RegExpPatterns {
+  /**
+   * Gets a pattern that matches common top-level domain names in lower case.
+   */
+  string commonTLD() {
```

Comment:
This name is not really in line with our style guide. At the very least, it should be `Tld` not `TLD`. Also, since this function returns a result, it should perhaps start with `get`.

Reply:
:heart: :+1: 

> it should perhaps start with get.   

And since it has multiple results it should probably start with `getA`. So `getACommonTld()`.